pkg/handler/handler.go

```diff
@@ -289,11 +290,13 @@ func (h handler) provision(w http.ResponseWriter, r *http.Request, params map[st
 			writeResponse(w, status, broker.ErrorResponse{Description: err.Error()})
 			return
 		}
+
+		username = userInfo.Username
```

Comment:
I would suggest that you may need to add logic because it could be the case that we do not have `Username` but `UID` from the `UserInfo` of the requesting user.